boards/nucleo-l1/dist/openocd.cfg

```diff
@@ -3,4 +3,8 @@ source [find interface/stlink-v2-1.cfg]
 
 transport select hla_swd
 
-source [find target/stm32l.cfg]
+if [catch {find target/stm32l.cfg}] {
```

Comment:
Can you add a comment above the if block to future users about why we are trying two different file names? Either reference this PR or a short textual description.